Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -213,6 +214,45 @@ export function runQTest(args: QTestArguments): Result {
         )
     );
     const qTestLogsDir: StaticDirectory = result.getOutputDirectory(logDir);
+
+    // If code coverage is enabled, start a pip that will perform coverage file upload.
+    if (qCodeCoverageEnumType === "DynamicCodeCov"){
+        let coverageLogDir = args.coverageLogsDir || Context.getNewOutputDirectory("coverageLogs");
+        let coverageConsolePath = p`${coverageLogDir}/coverageUpload.stdout`;

```

Comment:
so office build will place the coverageLogDir in object dir by default?

Reply:
Yes.